CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 - Upgrade NVIDIA driver to version 470.82.01.
 - Upgrade CUDA library to version 11.4.3.
 - Upgrade NVIDIA Fabric manager to version 470.82.01.
+- Upgrade Intel MPI Library to 2021.4.0.441.
```

Comment:
We are also setting the QT version for Intel MPI to 5.15.2.
Is it worth it to mention in the changelog?

Reply:
I don't think so it's just a third party dependency installed by Intel MPI itself.